diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/LineLength.kt

```diff
@@ -138,7 +155,30 @@ class LineLength(configRules: List<RulesConfig>) : DiktatRule(
                 else -> parent = parent.treeParent
             }
         } while (parent.treeParent != null)
-        return LongLineFixableCases.None
+        return None()
+    }
+
+    /**
+     * Analyzes the Binary expression and decides to go higher level with the analysis or analyze at this level
+     */
+    private fun isConditionToUpAnalysisBinExpression(parent: ASTNode, offset: Int): Boolean {
+        val listParentSearchInThisType = listOf(BINARY_EXPRESSION, PARENTHESIZED)
```

Comment:
May be we can extract into the variables the conditions and not the lists?
i.e. instead of 
```kotlin
val listParentSearchInThisType = listOf(BINARY_EXPRESSION, PARENTHESIZED)
```
use 
```kotlin
val condition1 = parent.treeParent.elementType in listOf(BINARY_EXPRESSION, PARENTHESIZED)
```
where the name of **condition1** will describe what's for it is
and so on

and then 
```kotlin
return cond1 || cond2 && cond3 ......
```